client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-definitions/setup/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Task } from '@automattic/launchpad';
+import { addQueryArgs } from '@wordpress/url';
+import { recordTaskClickTracksEvent } from '../../tracking';
+import { TaskAction } from '../../types';
+
+export const getSetupFreeTask: TaskAction = ( task, flow, context ): Task => {
+	const { siteInfoQueryArgs } = context;
+
+	return {
+		...task,
+		actionDispatch: () => recordTaskClickTracksEvent( task, flow, context ),
```

Comment:
It feels like we should be adding this in `getTaskDefinition()` rather than duplicating the logic for every task type.